libobs/obs-nix.c

```diff
@@ -633,9 +640,9 @@ static inline void add_key(obs_hotkeys_platform_t *context, obs_key_t key,
 
 	if (context->keycodes[key].list.num > 1) {
 		blog(LOG_DEBUG, "found alternate keycode %d for %s "
-		                "which already has keycode %d",
-		                code, obs_key_to_name(key),
-		                (int)context->keycodes[key].list.array[0]);
+				"which already has keycode %d",
+				code, obs_key_to_name(key),
+				(int)context->keycodes[key].list.array[0]);
```

Comment:
This whitespace change is not relevant to the commit nor needed and should be removed.